### DISCUSSION TITLE
xmake run 命令可以显示使用哪些环境变量吗？

### DISCUSSION THREAD

**Discussion by ThzShun** [xmake-io/xmake-repo#2412] [General]
主要想查看引入的包，是在什么环境变量下被调用，包括qt框架的 和 自定义包的。

**Comment by waruqi** [comment]
before_run 里面自己 print(os.getenvs()) 下

**Reply by ThzShun** [reply]
好的，谢谢。
查看后，有包引入的`AppData\Local`路径，但是看不到Qt相关的路径，这是哪个阶段设置的吗？还是说调用了什么批处理？
